README.md

```diff
@@ -124,7 +113,7 @@ There used to be one, but it's now decommissioned. You can spawn a new instance
 1. Edit your tasks to point to the [dev worker group](https://dxr.mozilla.org/build-central/rev/e2e751bce7198d358725904a9130bbb06a26c0f9/puppet/modules/pushapk_scriptworker/manifests/settings.pp#9).
```

Comment:
I think this section may want to be adjusted. Given that level-1 jobs can use the dev worker, I'm not sure we want to be suggesting testing changes with that workertype.

Reply:
Done